src/platform/echarts/wordcloud/echarts-wordcloud.js

```diff
@@ -0,0 +1,16556 @@
+/*!
+ * echarts-wordcloud
```

Comment:
Hi @felipeik, instead of copying over the built version like this, can you please do a submodule with the code that hasn't been built. Let us know if you have questions around this.

Reply:
I do have a question around this. Is there a similar case in covalent or covalent-echarts that I can compare to?